public/app/features/transformers/editors/CalculateFieldTransformerEditor.tsx

```diff
@@ -356,6 +360,78 @@ export class CalculateFieldTransformerEditor extends React.PureComponent<
     );
   }
 
+  //---------------------------------------------------------
+  // Unary Operator
+  //---------------------------------------------------------
+
+  updateUnaryOptions = (v: UnaryOptions) => {
+    const { options, onChange } = this.props;
+    onChange({
+      ...options,
+      mode: CalculateFieldMode.UnaryOperation,
+      unary: v,
+    });
+  };
+
+  onUnaryOperationChanged = (v: SelectableValue<string>) => {
+    const { unary } = this.props.options;
+    this.updateUnaryOptions({
+      ...unary!,
+      operator: v.value! as UnaryOperationID,
```

Comment:
We can avoid this `as` by using `SelectableValue<UnaryOperationId>`, adding a `unaryOperationID` property to the `UnaryOperationInfo` and using that property when we are constructing the options for the select.

Reply:
Thank you! Will modify!

Reply:
Modified this for both unary and binary editors!